### PR TITLE
DI-123 Connect to NHS.UK Azure Service Bus

### PR DESCRIPTION
**Pull request by stefaniuk**
This ticket is to connect to the NHS UK Azure Service Bus
Changes:

- Added a get_message.py file which gets messages from Azure Service Bus
- Added a put_message.py file to add messages to the Azure Service Bus
- A make target to run the both python files

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -56,3 +56,11 @@ kafka-producer-run:
 				--bootstrap-server kafka:9092 \
 				--topic TestTopic \
 		"
+
+python-get-message-run:
+	eval "$$(make secret-fetch-and-export-variables NAME=uec-di/deployment)"
```
Comment (stefaniuk) [inline Makefile:61]: Please, can we consider naming it `uec-dos-int-dev/deployment` to match our naming convention?

File: Makefile [hunk 2]
```diff
@@ -56,3 +56,11 @@ kafka-producer-run:
...
+	python application/get_message.py
+
+python-put-message-run:
+	eval "$$(make secret-fetch-and-export-variables NAME=uec-di/deployment)"
```
Comment (stefaniuk) [inline Makefile:65]: Please, can we consider naming it `uec-dos-int-dev/deployment` to match our naming convention?

**Review by JackPlowman — DISMISSED** [review]

**Review by nat-darke-nhs — APPROVED** [review]

File: .github/workflows/code-quality.yml [hunk 3]
```diff
@@ -20,6 +20,10 @@ jobs:
       - name: Check Terraform files format
         run: |
           build/automation/etc/githooks/scripts/terraform-format-pre-commit.sh
+      - name: Create coverage reports
+        run: |
+          pip install -r application/requirements.txt
```
Comment (stefaniuk) [inline .github/workflows/code-quality.yml:25]: Can we change that line to `make python-requirements`?

File: Makefile [hunk 4]
```diff
@@ -56,3 +56,24 @@ kafka-producer-run:
...
+python-peek-message-run:
+	eval "$$(make secret-fetch-and-export-variables NAME=uec-dos-int-dev/deployment)"
+	python application/peek_message.py
+
+python-put-message-run:
+	eval "$$(make secret-fetch-and-export-variables NAME=uec-dos-int-dev/deployment)"
+	python application/put_message.py
+
+python-pytest:
+	python -m pytest application/tests -s -x -v
```
Comment (stefaniuk) [inline Makefile:73]: Can we change that line to `make docker-run-tool CMD="python -m pytest ./tests -s -x -v" DIR=./application`

File: Makefile [hunk 5]
```diff
@@ -18,7 +18,7 @@ restart: stop start # Restart project
 log: project-log # Show project logs
 
 python-requirements:
-	make docker-run-python \
+	make docker-run-tools \
```
Comment (stefaniuk) [inline Makefile:21]: Can we change that line to `make docker-run-tool CMD="pip install -r requirements.txt" DIR=./application`?

**Review by nat-darke-nhs — APPROVED** [review]

**Review by nat-darke-nhs — APPROVED** [review]

**Review by mabe13 — APPROVED** [review]
Had a quick look over LGTM 👍

**Pull request merged**